src/elements/walletConnect/WalletConnectButton.tsx

```diff
@@ -1,26 +1,26 @@
 import { UseWalletConnect } from './useWalletConnect';
 import { shortenString } from 'utils/pureFunctions';
-import { Image } from 'components/image/Image';
 import { ReactComponent as IconWallet } from 'assets/icons/wallet.svg';
+import useENS from 'hooks/useENS';
+import Davatar from '@davatar/react';
 
 export const WalletConnectButton = ({
   handleWalletButtonClick,
   account,
   selectedWallet,
 }: UseWalletConnect) => {
-  const buttonText = account ? shortenString(account) : 'Connect Wallet';
+  const { ensName } = useENS(account || '');
+  const buttonText = account
+    ? ensName || shortenString(account)
+    : 'Connect Wallet';
 
   return (
     <button
       className="flex items-center md:btn-outline-secondary md:btn-sm md:mr-40"
       onClick={() => handleWalletButtonClick()}
     >
       {selectedWallet && account ? (
-        <Image
-          src={selectedWallet.icon}
-          alt="Wallet Logo"
-          className="w-[22px]"
-        />
+        <Davatar size={20} address={account} generatedAvatarType="jazzicon" />
```

Comment:
What image will be shown when a user without ens name logs in? We need to handle that case and show the selected wallet icon accordingly

Reply:
It will show the default jazzicon for that address

<img width="93" alt="Screenshot 2021-12-22 at 4 07 43 PM" src="https://user-images.githubusercontent.com/69431456/147079685-0626b6e9-a809-465e-bbc7-0abf00722071.png">